Index: gui-agent-linux/appvm-scripts/etc/X11/Xsession.d/25xdg-qubes-settings
===================================================================
--- gui-agent-linux.orig/appvm-scripts/etc/X11/Xsession.d/25xdg-qubes-settings
+++ gui-agent-linux/appvm-scripts/etc/X11/Xsession.d/25xdg-qubes-settings
@@ -22,4 +22,9 @@ if [ -x /usr/bin/xsettingsd ]; then
     /usr/bin/xsettingsd &
 fi

+if [ "$XDG_DATA_DIRS" = "" ]; then
+    XDG_DATA_DIRS='/usr/share/ubuntu/:/usr/share/gnome/:/usr/local/share/:/usr/share/'
+    export XDG_DATA_DIRS
+fi
+
 fi
\ No newline at end of file
